src/app/shared-module/title-fades/title-fades.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-title-fades',
+  templateUrl: './title-fades.component.html',
+  styleUrls: ['./title-fades.component.scss'],
+})
+export class TitleFadesComponent implements OnInit {
+  constructor() {}
+
+  @Input() color: string = '';
+  @Input() title: string = '';
+
+  isYellow: boolean = false;
+  isOrange: boolean = false;
+
+  ngOnInit() {
+    if (this.color == 'yellow') {
```

Comment:
You don’t have to do this check and set, you can check in the inputs if the string is yellow or orange. Then use directly them in the template directive, no need of two more variables

Reply:
I agree. Done.